core/runtime/wabt/instrument.hpp

```diff
@@ -28,12 +28,12 @@ namespace kagome::runtime {
    * - set memory limit
    * https://github.com/paritytech/polkadot-sdk/blob/11831df8e709061e9c6b3292facb5d7d9709f151/substrate/client/executor/wasmtime/src/runtime.rs#L651
    */
-  WabtOutcome<common::Buffer> prepareBlobForCompilation(
+  WabtOutcome<common::Buffer> instrumentCodeForCompilation(
```

Comment:
Revert, this name is from rust
```suggestion
  WabtOutcome<common::Buffer> prepareBlobForCompilation(
```